test/known_issues/test-vm-ownkeys.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+require('../common');
+const vm = require('vm');
+const assert = require('assert');
+
+const sym1 = Symbol('1');
+const sym2 = Symbol('2');
+const sandbox = {
+  a: true,
+  [sym1]: true
+};
+Object.defineProperty(sandbox, 'b', { value: true });
+Object.defineProperty(sandbox, sym2, { value: true });
+
+const ctx = vm.createContext(sandbox);
+
+// sanity check
```

Comment:
Could you make a comment explicitly showing which assertion you expect to fail? Should we split this in multiple files so we're aware if one of the problems gets fixed? If anything else breaks? Actually, the sanity checks should probably not be here or commented out. Since the test runner cannot distinguish between good and bad asserts in known_issues.

Reply:
OK, thanks. I split this in 3 tests and commented out the sanity checks.

Reply:
If the sanity checks are going to be added back once this is no longer a known issue, can you add another comment to say that.

Reply:
sure